kudu/src/test/java/docs/javadsl/KuduTableTest.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package docs.javadsl;
+
+import akka.Done;
+import akka.NotUsed;
+import akka.actor.ActorSystem;
+import akka.japi.Pair;
+import akka.stream.ActorMaterializer;
+import akka.stream.Materializer;
+import akka.stream.alpakka.kudu.KuduTableSettings;
+import akka.stream.alpakka.kudu.javadsl.KuduTable;
+import akka.stream.javadsl.Flow;
+import akka.stream.javadsl.Keep;
+import akka.stream.javadsl.Sink;
+import akka.stream.javadsl.Source;
+import akka.testkit.javadsl.TestKit;
+import org.apache.kudu.ColumnSchema;
+import org.apache.kudu.Schema;
+import org.apache.kudu.Type;
+import org.apache.kudu.client.CreateTableOptions;
+import org.apache.kudu.client.KuduClient;
+import org.apache.kudu.client.KuduException;
+import org.apache.kudu.client.PartialRow;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+
+public class KuduTableTest {
+  private static final String kuduMasterAddresses = "127.0.0.1:7051";
+
+  private static ActorSystem system;
+  private static Materializer materializer;
+  private static Schema schema;
+
+  private static KuduClient kuduClient;
+  private static KuduTableSettings<Person> tableSettings;
+
+  @BeforeClass
+  public static void setup() {
+    system = ActorSystem.create();
+    materializer = ActorMaterializer.create(system);
+
+    // #configure
+    // Kudu Client
+    kuduClient = new KuduClient.KuduClientBuilder(kuduMasterAddresses).build();
+    system.registerOnTermination(
+        () -> {
+          try {
+            kuduClient.shutdown();
+          } catch (KuduException e) {
+            e.printStackTrace();
+          }
+        });
```

Comment:
Good. This management of the client on shutdown could be done automatically in an extension.